features/canvas-reset.yml

```diff
@@ -0,0 +1,6 @@
+name: reset()
```

Comment:
I find 10 `reset()` methods in webref, so this will need to be disambiguated. "Canvas `reset()`" would be most similar to "Canvas text baselines". I guess let's also do the same for all of these features?